### PR TITLE
Feature#44/shop view

### PR DESCRIPTION
**Pull request by nallely64**
# Description

 - What?: I've added the layout for history shopping view
 - Why?: To show the history of user's purchases 

  ## Screenshots 
Captura de pantalla de 2022-05-19 21-16-49

### CODE REVIEW

File: app/assets/stylesheets/components/_purchases_card.scss [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    margin: 0 auto;
+    max-width: 700px;
+}
+
+.purchases_title::after {
+    content: '';
+    background-color: $icon-yellow;
+    width: 160px;
+    height: 3px;
+    display: block;
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_purchases_card.scss:11]: Sort alphabetically this block.

File: app/assets/stylesheets/components/_purchases_card.scss [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+}
+
+.purchases_cards {
+    display: flex;
+    flex-direction: column;
+    margin: 0 auto;
+    height: fit-content;
+    width: fit-content;
+    padding: 20px;
+    line-height: 50px;
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_purchases_card.scss:21]: Sort alphabetically this block.

File: app/assets/stylesheets/components/_purchases_card.scss [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+.purchases_card {
+    height: 200px;
+    width: 700px;
+    padding: 20px;
+    box-sizing: border-box;
+    border: 2px solid $red;
+    border-radius: 10px;
+    position: relative;
+}
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_purchases_card.scss:37]: Sort alphabetically this block.

File: app/assets/stylesheets/components/_purchases_card.scss [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+
+.purchases_card__body {
+    padding: 2px;
+
+    .buttons__evaluate {
+        position: absolute;
+        right: 0;
+        max-width: 230px;
+        top: 20px;
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_purchases_card.scss:46]: Sort alphabetically this block.

File: app/assets/stylesheets/components/_purchases_card.scss [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+
+
+.purchases_cards__button {
+    background-color: $button-yellow;
+    border-radius: 20px;
+    border: $button-yellow;
+    height: 45px;
+    text-align: center;
+    width: 180px;
+    margin: 15px;
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_purchases_card.scss:59]: Sort alphabetically this block.

File: app/assets/stylesheets/components/_purchases_card.scss [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+    margin-left: 0;
+}
+
+#evaluate_product {
+    background-color: $profile-button;
+    border: $profile-button;
+}
+
+#evaluate_seller
+{
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_purchases_card.scss:76]: fix the position of the `{`

File: app/assets/stylesheets/components/_purchases_card.scss [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+}
+
+#purchases_second_button {
+    color: $white;
+    margin: 10px;
+}
+
+.purchases_card__image {
+    width: 135px;
+    height: 135px;
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_purchases_card.scss:97]: Sort alphabetically this block.

File: app/assets/stylesheets/components/_purchases_card.scss [hunk 8]
```diff
@@ -0,0 +1,130 @@
...
+    > img {
+        object-fit: cover;
+    }
+}
+
+.purchases_title {
+    margin-top: 70px;
+    font-size: 30px;
+    color: $purchases-title;
+    margin-bottom: 2px;
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_purchases_card.scss:108]: Sort alphabetically this block.

File: app/assets/stylesheets/components/_purchases_card.scss [hunk 9]
```diff
@@ -0,0 +1,130 @@
...
+}
+
+.purchases_card__title {
+    margin: 0px;
+}
+
+.purchases_card__quantity {
+    margin-top: 0px;
+    font-size: 14px;
+    color: $gray-dark;
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_purchases_card.scss:118]: Sort alphabetically this block.

File: app/assets/stylesheets/components/_purchases_card.scss [hunk 10]
```diff
@@ -0,0 +1,130 @@
...
+}
+
+.purchases_card__subtitle, .mb-2, .text-muted {
+    margin: 0px;
+    color: $gray;
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_purchases_card.scss:123]: Sort alphabetically this block.

File: app/assets/stylesheets/components/_purchases_card.scss [hunk 11]
```diff
@@ -0,0 +1,130 @@
...
+}
+
+.purchases_line {
+    width: 420px;
+    margin-top: 0px;
+    color: $black;
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_purchases_card.scss:129]: Sort alphabetically this block.

**Review by JuanequeX — CHANGES_REQUESTED** [review]
Please review your identation in the `_purchases_card.scss` file, it should be have 2 spaces.

File: app/assets/stylesheets/components/_purchases_card.scss (hunk 1 above)
Comment (nallely64) [reply app/assets/stylesheets/components/_purchases_card.scss]: done!

File: app/assets/stylesheets/components/_purchases_card.scss (hunk 2 above)
Comment (nallely64) [reply app/assets/stylesheets/components/_purchases_card.scss]: done!

File: app/assets/stylesheets/components/_purchases_card.scss (hunk 3 above)
Comment (nallely64) [reply app/assets/stylesheets/components/_purchases_card.scss]: done!

File: app/assets/stylesheets/components/_purchases_card.scss (hunk 4 above)
Comment (nallely64) [reply app/assets/stylesheets/components/_purchases_card.scss]: done.

File: app/assets/stylesheets/components/_purchases_card.scss (hunk 5 above)
Comment (nallely64) [reply app/assets/stylesheets/components/_purchases_card.scss]: done.

File: app/assets/stylesheets/components/_purchases_card.scss (hunk 6 above)
Comment (nallely64) [reply app/assets/stylesheets/components/_purchases_card.scss]: done.

File: app/assets/stylesheets/components/_purchases_card.scss (hunk 7 above)
Comment (nallely64) [reply app/assets/stylesheets/components/_purchases_card.scss]: done.

File: app/assets/stylesheets/components/_purchases_card.scss (hunk 8 above)
Comment (nallely64) [reply app/assets/stylesheets/components/_purchases_card.scss]: done.

File: app/assets/stylesheets/components/_purchases_card.scss (hunk 9 above)
Comment (nallely64) [reply app/assets/stylesheets/components/_purchases_card.scss]: done.

File: app/assets/stylesheets/components/_purchases_card.scss (hunk 10 above)
Comment (nallely64) [reply app/assets/stylesheets/components/_purchases_card.scss]: done.

File: app/assets/stylesheets/components/_purchases_card.scss (hunk 11 above)
Comment (nallely64) [reply app/assets/stylesheets/components/_purchases_card.scss]: done, than you!

**Review by germanruzca — DISMISSED** [review]
LGTM

**Review by juperameza — APPROVED** [review]
LGTM!!

**Review by JuanequeX — APPROVED** [review]
LGTM

**Review by germanruzca — DISMISSED** [review]

**Review by juperameza — DISMISSED** [review]
LGTM

**Review by germanruzca — APPROVED** [review]

**Review by juperameza — APPROVED** [review]
LGTM

**Pull request merged**